qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -151,8 +176,10 @@ setup()
     trap cleanup INT TERM EXIT
 
     if [ -n "${RBD_MIRROR_TEMDIR}" ]; then
+	test -d "${RBD_MIRROR_TEMDIR}" ||
 	mkdir "${RBD_MIRROR_TEMDIR}"
 	TEMPDIR="${RBD_MIRROR_TEMDIR}"
+	cd ${TEMPDIR}
```

Comment:
Yes, without this change, when you tried to use  RBD_MIRROR_USE_EXISTING_CLUSTER locally in order to rerun the tests without restarting the clusters, it did not work, because if this variable was not set we did not change the working directory to TEMPDIR (for teuthology it was OK).